crates/wash-cli/src/ctx/mod.rs

```diff
@@ -317,6 +317,19 @@ fn prompt_for_context() -> Result<WashContext> {
         &Some(DEFAULT_NATS_TIMEOUT_MS.to_string()),
     )?;
 
+    let ctl_tls_first = match prompt_for_choice(
+        &StringEntry {
+            default: Some("false".to_string()),
+            choices: Some(vec!["true".to_string(), "false".to_string()]),
+            regex: None,
+        },
+        "Should the control interface use TLS first?",
+    ) {
+        Ok(0) => Some(true),
+        Ok(1) => Some(false),
+        _ => None,
+    };
+
```

Comment:
Would it make sense to try to detect whether the user is trying to use TLS before asking this? It's a bit difficult because there are many different ways they could express wanting to use TLS, but it would be nice if we could make this question optional.

Reply:
Should I judge based on the provided host url, e.g. see if it starts with `tls`?

Reply:
Yeah, so it's kind of tricky -- that, and whether the a custom CA file has been provided, from what I can see.

The perfect changeset might be to *ask* the person if their NATS connection uses TLS (and then do a whole bunch of other prompting), but that can be outside the scope of this change! Better to get some support in there and then improve it later if it's a stumbling block for people

Reply:
Hmnn, let's get this in here, and maybe re-think this in the future!

Reply:
Sorry, I've been a bit busy these days so I may not have time to address this very soon.